### ISSUE TITLE
Unable to access LazyLoad setting from extensions applied to Image

### ISSUE THREAD

**Issue by lozcalver** [silverstripe/silverstripe-assets#513]
Example:

```php
<?php

namespace App\Extensions\Assets;

use SilverStripe\Core\Extension;

class ImageExtension extends Extension
{
    public function Test()
    {
        var_dump($this->getOwner()->IsLazyLoaded());
    }
}
```

Calling `$Image.LazyLoad(false).Test` in a template will always output `true`.

This seems to be because the `ImageManipulation` trait is applied to both `File` (`Image` extends `File`) and `DBFile`. Calling `.LazyLoad(false)` calls the following code:

https://github.com/silverstripe/silverstripe-assets/blob/ce9464b6282c34e37324987ea2dd74cae89c50ca/src/ImageManipulation.php#L1101-L1111

This then returns the a `DBFile` instance with the attribute set, but the attribute is never set on the `Image` object itself. Setting the attributes on the `Image` object via `$file->getFailover()->initAttributes()` fixes this, but goes against the documentation for `ImageManipulation::setAttribute()` which states the method is supposed to be immutable

### PRs
- https://github.com/silverstripe/silverstripe-assets/pull/514

**Comment by GuySartorelli** [comment]
I'm just trying to wrap my head around the implications of this - do you have a more realistic example of where this would cause a problem?

**Comment by lozcalver** [comment]
> I'm just trying to wrap my head around the implications of this - do you have a more realistic example of where this would cause a problem?

So my use case is [this module](https://github.com/kinglozzer/silverstripe-picture), the lazyload setting currently isn’t copied through so I can’t enable/disable `loading=lazy` for specific images, only globally. Any extension that needs to know if an image is lazy loaded or not would be affected though

**Comment by GuySartorelli** [comment]
Fixed in [1.12.1](https://github.com/silverstripe/silverstripe-assets/releases/tag/1.12.1)
